<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convertor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 233, 79);
color: rgb(32, 74, 135);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="combo_kind">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Weight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bytes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lenght</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Temperature</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kind</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>from:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>to:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_from">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_to">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_res">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_inp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clc">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>507</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
